_posts/2020-12-18-community-meeting-poll.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: posts
+author: Kyle Davis
+comments: true
+title: "2021 Open Distro for Elasticsearch Community Meetings"
+categories:
+- meetings
+feature_image: "https://d2908q01vomqb2.cloudfront.net/ca3512f4dfa95a03169c5a670a4c91a19b3077b4/2019/03/26/open_disto-elasticsearch-logo-800x400.jpg"
+---
+
+We've been having regular bi-weekly community meetings over Zoom at 10 am Pacific on Mondays for quite some time.  While there has certainly been good response and attendance, we recognize that this time could be less than ideal for those who live in timezones with a UTC greater than +4 (India and points west to the international dateline).
```

Comment:
Recommend "10:00 AM Pacific" (or PT if you prefer)

Reply:
I'd try to be more specific and call out our broad userbase here. Something like, "we recognize that this time isn't ideal for our community members in India, Russia, Japan, South Korea, Australia, and everyone else located closer to the west side of the International Date Line."